_layouts/default.html

```diff
@@ -51,12 +51,11 @@ <h1 class="noselect">South Eugene Robotics Team</h1>
             </li>
             {% endif %}
             {% endfor %}
-            <li style="height: 36px">
-                <form action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank" style="margin-top: 5px">
+            <li>
+                <form name="donateform" action="https://www.paypal.com/cgi-bin/webscr" method="post" target="_blank" style="margin: 0">
                     <input type="hidden" name="cmd" value="_s-xclick">
                     <input type="hidden" name="hosted_button_id" value="XAPA9VJHWAE94">
-                    <input type="image" src="https://www.paypalobjects.com/en_US/i/btn/btn_donate_LG.gif" border="0" name="submit" title="Donate with PayPal" alt="Donate with PayPal button">
-                    <img alt="" border="0" src="https://www.paypal.com/en_US/i/scr/pixel.gif" width="1" height="1">
+                    <a class="donate" onclick="document.getElementsByName('donateform')[0].submit()">Donate</a>
```

Comment:
Technically not valid HTML (and won't apply <a> styles, like `cursor: pointer`) unless there's an `href` set. Also switch to id for simplicity (see above).

```suggestion
                    <a href="#" class="donate" onclick="document.getElementsById('donateform').submit()">Donate</a>
```

Reply:
Done 👍